cmd/neofs-node/container.go

```diff
@@ -8,8 +8,13 @@ import (
 	"errors"
 	"fmt"
 
-	containerV2 "github.com/nspcc-dev/neofs-api-go/v2/container"
-	containerGRPC "github.com/nspcc-dev/neofs-api-go/v2/container/grpc"
+	apicontainer "github.com/nspcc-dev/neofs-api-go/v2/container"
+	protocontainer "github.com/nspcc-dev/neofs-api-go/v2/container/grpc"
+	apirefs "github.com/nspcc-dev/neofs-api-go/v2/refs"
+	refs "github.com/nspcc-dev/neofs-api-go/v2/refs/grpc"
```

Comment:
should this be `protorefs`? just to make it meaningful according to the other aliases

Reply:
there is no `refs` package in SDK root level, so we can do w/o alias (considering that apigo will leave)